### ISSUE TITLE
multiple defintion of numNets in SPRoute

### ISSUE THREAD

**Issue by ddn0** [IntelligentSoftwareSystems/Galois#221]
Only affects Arch builds at the moment.

```
/usr/sbin/ld: CMakeFiles/SPRoute.dir/bookshelf_IO.c.o:(.bss+0xdc): multiple definition of `numNets'; /usr/sbin/ld: DWARF error: could not find variable specification at offset 6dcfc
...
CMakeFiles/SPRoute.dir/main.cpp.o:/root/project/lonestar/eda/cpu/SPRoute/DataProc.h:38: first defined here
collect2: error: ld returned 1 exit status
```

https://app.circleci.com/pipelines/github/IntelligentSoftwareSystems/Galois/441/workflows/dd17a73e-48cd-475f-a097-1f4408c49a62/jobs/8265/steps

**Comment by Michael-JY-He** [comment]
Fixed. Please check again.

**Comment by ddn0** [comment]
> Fixed. Please check again.

Thanks for the quick turn around!

BTW, I think the easiest way to indicate that this issue is fixed would be to link to the PR that addresses it. Then, we'd be able to see that the previously failing CI run now is passing.

I can understand that sometimes that would be a bother so it is not super important to do all the time, but I thought I'd raise this option if you were curious.

**Comment by Michael-JY-He** [comment]
Yeah I'm glad to do that but how to link to the PR?

**Comment by ddn0** [comment]
You can:
- copy and paste the link as a comment
- click on the "Linked pull requests" widget on the right and select it
- write `#<number of the PR>` (e.g., #158)  (clicking "Preview" on the editor pane will let you see if your markdown syntax is being interpreted **properly)

**Comment by ddn0** [comment]
Fixed in c6cbdacdf352022194e4ed199bd375a8fb56f453
